tools/metrics/gauges.go

```diff
@@ -116,32 +117,41 @@ func AppCountGauge(c *Client, interval time.Duration) MetricReadCloser {
 		if err != nil {
 			return err
 		}
-		started := 0
-		stopped := 0
+
+		counters := map[string]map[bool]int{
```

Comment:
It would be nice to have a comment about what this structure means. e.g. counters[STATE][ORG_IS_TRIAL] = number off apps where state is STATE and ORG_IS_TRIAL is true if the app is running in an org which is in trial mode.